src/components/ApplyForm/ApplyForm.jsx

```diff
@@ -1,8 +1,139 @@
 import Button from "../../components/Buttton/Button";
+import { useState } from 'react'
+import { redirect, useNavigate } from 'react-router-dom';
+import Dropdown from "../Dropdown/Dropdown";
+import postCreateAccount from "../../api/post_create_account";
+import postLogin from "../../api/post_login";
 import "./ApplyForm.css";
-import Dropdown from "../../components/Dropdown/Dropdown";
+
 
 const ApplyForm = () =>{
+    const navigate = useNavigate();
+
+
+    const [errorMessage, setErrorMessage] = useState("")
+    const [formInvalid, setFormInvalid] = useState("")
+    const [checkedState, setCheckedState] = useState([]);
+    const [mentored, setMentored] = useState(true);
+
+    // until we have a separate get request for skills list:
+    const skills = [ "Python", "Django", "DRF", "React", "Javascript", "Frontend",  "Backend", "HTML-CSS"]
```

Comment:
Why are we hard coding this instead of pulling from the GET request?

Reply:
Fixed up in https://github.com/SheCodesAus/layer_8_victory_frontend_bris_2023/pull/13/commits/7e5f1a9d95ffe54cd820da247c92905053f05fe5